convert2rhel/unit_tests/redhatrelease_test.py

```diff
@@ -31,13 +32,23 @@
 
 from convert2rhel import unit_tests  # Imports unit_tests/__init__.py
 from convert2rhel import pkgmanager, redhatrelease, utils
-from convert2rhel.redhatrelease import YumConf
+from convert2rhel.redhatrelease import YumConf, get_os_release_filepath, get_system_release_filepath
 from convert2rhel.systeminfo import system_info
-from convert2rhel.toolopts import tool_opts
 
 
-class TestRedHatRelease(unittest.TestCase):
+YUM_CONF_WITHOUT_DISTROVERPKG = """[main]
+installonly_limit=3
+
+#  This is the default"""
+
+YUM_CONF_WITH_DISTROVERPKG = """[main]
+installonly_limit=3
+distroverpkg=centos-release
+
+#  This is the default"""
```

Comment:
Are both the default?

Reply:
I didn't change those strings, just moved them to the top of the file for better organizing. They were already like this

Reply:
Ah I see it now, no worries